spiceaidocs/docs/components/data-connectors/index.md

```diff
@@ -11,62 +11,66 @@ Data Connectors provide connections to databases, data warehouses, and data lake
-| `mssql`         | MS SQL Server | Alpha  | Tabular Data Stream (TDS)           | `full`           | ❌               | ❌                |
-| `mysql`         | MySQL         | Alpha  |                                     | `full`           | ❌               | ❌                |
-| `odbc`          | ODBC          | Alpha  | ODBC                                | `full`           | ❌               | ❌                |
-| `postgres`      | PostgreSQL    | Alpha  |                                     | `full`           | ❌               | ❌                |
-| `sharepoint`    | SharePoint    | Alpha  |                                     | `full`           | ❌               | ✅                |
-| `snowflake`     | Snowflake     | Alpha  | Arrow                               | `full`           | ❌               | ❌                |
-| `spiceai`       | Spice.ai      | Alpha  | Arrow Flight                        | `append`, `full` | ✅               | ❌                |
-| `s3`            | S3            | Alpha  | Parquet, CSV                        | `full`           | ❌               | ✅                |
-| `abfs`            | Azure BlobFS            | Alpha  | Parquet, CSV                        | `full`           | ❌               | ✅                |
-| `sharepoint`    | SharePoint    | Alpha  |                                     | `full`           | ❌               | ✅                |
-| `spark`         | Spark         | Alpha  | Spark Connect                       | `full`           | ❌               | ❌                |
+| Name            | Description   | Status | Protocol/Format                     | Refresh Modes               | Supports Ingestion | Supports Documents |
+| --------------- | ------------- | ------ | ----------------------------------- | --------------------------- | ------------------ | ------------------ |
+| `abfs`          | Azure BlobFS  | Alpha  | Parquet, CSV                        | `append`, `full`            | Roadmap            | ✅                 |
+| `clickhouse`    | Clickhouse    | Alpha  |                                     | `append`, `full`            | ❌                 | ❌                 |
+| `databricks`    | Databricks    | Beta   | Spark Connect <br/> S3 / Delta Lake | `append`, `full`            | Roadmap            | ❌                 |
+| `debezium`      | Debezium      | Alpha  | CDC, Kafka                          | `append`, `full`, `changes` | ❌                 | ❌                 |
+| `delta_lake`    | Delta Lake    | Beta   | Delta Lake                          | `append`, `full`            | Roadmap            | ❌                 |
+| `dremio`        | Dremio        | Alpha  | Arrow Flight SQL                    | `append`, `full`            | ❌                 | ❌                 |
+| `file`          | File          | Beta   | Parquet, CSV                        | `append`, `full`            | Roadmap            | ✅                 |
```

Comment:
Is the file connector Beta? I don't recall us testing it from the old Beta criteria.